projects/fourier/test/fourier/services/test/tuist/unit_test.rb

```diff
@@ -20,7 +20,7 @@ def test_call
             Utilities::System
               .expects(:system)
               .with("swift", "test")
-
+            
```

Comment:
you should remove this as well.
Aslo, pay attention to not leaving empty spaces in empty line or linting will fail